packages/tools/devtools/devtools-example/src/FluidObject.ts

```diff
@@ -0,0 +1,162 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
+import { IFluidHandle, IFluidLoadable } from "@fluidframework/core-interfaces";
+import { SharedTree } from "@fluid-experimental/tree";
+import { SharedCounter } from "@fluidframework/counter";
+import { SharedString } from "@fluidframework/sequence";
+import { SharedCell } from "@fluidframework/cell";
+import { SharedMatrix } from "@fluidframework/matrix";
+
+/**
+ * CollaborativeDevtoolExampleAppObjects uses the React CollaborativeTextArea to load a collaborative HTML <textarea>
+ */
+export class CollaborativeDevtoolExampleAppObjects extends DataObject {
```

Comment:
What do you think about just calling this something like "AppData"?

Reply:
Yeah. Sounds much better! Renamed it!